.github/actions/install-aiida-core/action.yml

```diff
@@ -29,19 +31,18 @@ runs:
       python-version: ${{ inputs.python-version }}
 
   - name: Set up uv
-    uses: astral-sh/setup-uv@v4
+    uses: astral-sh/setup-uv@v5
     with:
-      version: 0.5.6
+      version: 0.5.x
+      python-version: ${{ inputs.python-version }}
 
   - name: Install dependencies from uv lock
     if: ${{ inputs.from-lock == 'true' }}
     # NOTE: We're asserting that the lockfile is up to date
-    # NOTE2: 'pre-commit' extra recursively contains other extras
-    # needed to run the tests.
-    run: uv sync --locked --extra pre-commit
```

Comment:
Previously we were not respecting the `inputs.extras` when using the lockfile. Now we do (at the expense of very ugly Github Actions syntax :-( )

Reply:
I think it is inspired from JavaScript so at least not completely random.

Reply:
~~in CI will `inputs.extras` in `uv sync --locked ${{ inputs.extras && format('--extra {0}', inputs.extras) || '' }}` always be `true`? Because if it is not pass as input it will set to "pre-commit" as default, no? Then this clause can be simplified.~~

Never mind, I see extra='' below.